lib/cfg-parser.c

```diff
@@ -421,6 +421,8 @@ cfg_process_flag(CfgFlagHandler *handlers, gpointer base, const gchar *flag_)
               else
                 *field = (*field) & ~handler->param;
               return TRUE;
+            default:
```

Comment:
Operation should be either SET or CLEAR. This should be g_assert_not_reached() again.